pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/mercury/ClinicalOrderDto.java

```diff
@@ -1,20 +1,55 @@
 package org.broadinstitute.dsm.db.dto.mercury;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import org.broadinstitute.dsm.db.dao.mercury.ClinicalOrderDao;
+import org.broadinstitute.dsm.statics.DBConstants;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
 
 @AllArgsConstructor
 @Getter
 public class ClinicalOrderDto {
 
-
+    @SerializedName("shortId")
```

Comment:
Down the road, we do plan on making a query for orders accessible to the frontend, so these annotations are here to help us do that in a subsequent sprint.